staramr/blast/results/pointfinder/PointfinderHitHSP.py

```diff
@@ -12,39 +12,37 @@
 
 class PointfinderHitHSP(AMRHitHSP):
 
-    def __init__(self, file, blast_record, hit, hsp):
+    def __init__(self, file, blast_record):
         """
         Creates a new PointfinderHitHSP.
         :param file: The input file.
         :param blast_record: The Bio.Blast.Record this hit came from.
-        :param hit: The particular Bio.Blast.Record.Alignment.
-        :param hsp: The particular Bio.Blast.Record.HSP.
         """
-        super().__init__(file, blast_record, hit, hsp)
+        super().__init__(file, blast_record)
 
-    def get_gene(self):
+    def get_amr_gene_name(self):
         """
         Gets the particular gene name for the PointFinder hit.
         :return: The gene name.
         """
-        return self.hit.hit_id
+        return self._blast_record['sseqid']
 
     def _get_match_positions(self):
-        return [i for i, c in enumerate(self.hsp.match) if c == ' ']
+        return [i for i, (x, y) in enumerate(zip(self._blast_record['sseq'], self._blast_record['qseq'])) if x != y]
 
-    def _get_mutation_positions(self, start, database_frame, query_frame):
-        return [CodonMutationPosition(i, self.hsp.sbjct, self.hsp.query, start, database_frame, query_frame) for i
+    def _get_mutation_positions(self, start, database_strand):
+        return [CodonMutationPosition(i, self._blast_record['sseq'], self._blast_record['qseq'], start, database_strand)
+                for i
```

Comment:
Just my formatting preference, but I like to put the `for <var> in <collection>` on the same line if possible just like you would have to for a regular `for` loop.

Reply:
I think this came from using the **Reformat** option in pyCharm.  I've changed this so that pyCharm will skip reformatting that line in 887d63ebb2c0db83593405b96816f3d5458abddb.

Reply:
Oh that's weird... Something to watch out for with PyCharm I guess!